model/src/lib.rs

```diff
@@ -44,20 +34,20 @@ pub struct Signature {
 #[derive(Eq, PartialEq, Clone, Copy, Debug, Default, Deserialize, Serialize)]
 #[serde(rename_all = "camelCase")]
 pub struct UserOrder {
```

Comment:
reorderd fields, changed to u256, changed fill_type from enum to bool partially_fillable because that's the same as in the smart contract